docs/developers/contributing/testing.md

```diff
@@ -259,13 +259,65 @@ def test_something_else(qtbot):
     ...
 ```
 
+(qt_viewer)=
+#### `qt_viewer` and `viewer_model`
+
+For the test that are using only `ViewerModel` api or are only checking 
+rendering of the viewer, since `napari==0.5.4` we have implemented the `qt_viewer` [pytest fixture](https://docs.pytest.org/en/stable/explanation/fixtures.html).
+For the current moment, it is only for internal use and is not exported to the global scope, 
+as it is defined in `conftest.py` file. 
+
+The `qt_viewer` fixture return the instance of {class}`~napari.qt.QtViewer` class.
+This class do not provide the same api as the {class}`~napari.ViewerModel` class, 
+but have associated {class}`~napari.ViewerModel` instance, which can be accessed by `viewer` attribute.
+Alternatively you could use `viewer_model` fixture, which return this instance of {class}`~napari.ViewerModel` class.
```

Comment:
```suggestion
Since `napari==0.5.4` we have implemented the `qt_viewer` [pytest fixture](https://docs.pytest.org/en/stable/explanation/fixtures.html) which can be used for tests that are only using the `ViewerModel` api or are only checking rendering of the viewer. 
For the current moment, it is only for internal use and is not exported to the global scope, 
as it is defined in `conftest.py` file. 

The `qt_viewer` fixture returns the instance of the {class}`~napari.qt.QtViewer` class.
This class does not provide the same api as the {class}`~napari.ViewerModel` class, 
but has an associated {class}`~napari.ViewerModel` instance, which can be accessed by the `viewer` attribute.
Alternatively, you could use the `viewer_model` fixture, which returns this instance of {class}`~napari.ViewerModel` class.
```